db/migrate/20220514154201_create_posts.rb

```diff
@@ -0,0 +1,12 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.string :title
+      t.text :text
+      t.integer :commentsCounter
+      t.integer :likesCounter
```

Comment:
- [ ] {Optional] In ruby language, the normal way to declare variables or methods is to use the snake case format. For example, `comments_counter`. 👍

Reply:
@Gambit142 Ok Thank you for the feedback. I will take it into consideration in subsequent commit and PR.